mouse/mouse.nim

```diff
@@ -1,34 +1,43 @@
-import os
-
+import std/[os, strutils, strformat]
 
 type
     Vec2D* = tuple
         x: int
         y: int
+    
+    MouseButton* = enum
```

Comment:
Instead of having different types for Windows/Linux we can have a unified enum type for the mouse button, and then convert it to what we need in the OS-specific procs